src/NUnitFramework/tests/Constraints/SubstringConstraintTests.cs

```diff
@@ -67,6 +67,88 @@ public void SetUp()
+    public class UsingTests : StringConstraintTests
+    {
+        [TestCase(" ss ", "�", false, StringComparison.CurrentCulture, true)]
+        [TestCase(" SS ", "�", false, StringComparison.CurrentCulture, false)]
+        [TestCase(" ss ", "s", false, StringComparison.CurrentCulture, true)]
+        [TestCase(" SS ", "s", false, StringComparison.CurrentCulture, false)]
+        [TestCase(" ss ", "�", false, StringComparison.CurrentCultureIgnoreCase, true)]
+        [TestCase(" SS ", "�", false, StringComparison.CurrentCultureIgnoreCase, true)]
+        [TestCase(" ss ", "s", false, StringComparison.CurrentCultureIgnoreCase, true)]
+        [TestCase(" SS ", "s", false, StringComparison.CurrentCultureIgnoreCase, true)]
+#if !NETSTANDARD1_3
+        [TestCase(" ss ", "�", false, StringComparison.InvariantCulture, true)]
+        [TestCase(" SS ", "�", false, StringComparison.InvariantCulture, false)]
+        [TestCase(" ss ", "s", false, StringComparison.InvariantCulture, true)]
+        [TestCase(" SS ", "s", false, StringComparison.InvariantCulture, false)]
+        [TestCase(" ss ", "�", false, StringComparison.InvariantCultureIgnoreCase, true)]
+        [TestCase(" SS ", "�", false, StringComparison.InvariantCultureIgnoreCase, true)]
+        [TestCase(" ss ", "s", false, StringComparison.InvariantCultureIgnoreCase, true)]
+        [TestCase(" SS ", "s", false, StringComparison.InvariantCultureIgnoreCase, true)]
+#endif
+        [TestCase(" ss ", "�", false, StringComparison.Ordinal, false)]
+        [TestCase(" SS ", "�", false, StringComparison.Ordinal, false)]
+        [TestCase(" ss ", "s", false, StringComparison.Ordinal, true)]
+        [TestCase(" SS ", "s", false, StringComparison.Ordinal, false)]
+        [TestCase(" ss ", "�", false, StringComparison.OrdinalIgnoreCase, false)]
+        [TestCase(" SS ", "�", false, StringComparison.OrdinalIgnoreCase, false)]
+        [TestCase(" ss ", "s", false, StringComparison.OrdinalIgnoreCase, true)]
+        [TestCase(" SS ", "s", false, StringComparison.OrdinalIgnoreCase, true)]
+
+        [TestCase(" ss ", "�", false, null, false)]
+        [TestCase(" SS ", "�", false, null, false)]
+        [TestCase(" ss ", "s", false, null, true)]
+        [TestCase(" SS ", "s", false, null, false)]
+
+        [TestCase(" ss ", "�", true, StringComparison.CurrentCulture, true)]
+        [TestCase(" SS ", "�", true, StringComparison.CurrentCulture, false)]
+        [TestCase(" ss ", "s", true, StringComparison.CurrentCulture, true)]
+        [TestCase(" SS ", "s", true, StringComparison.CurrentCulture, false)]
+        [TestCase(" ss ", "�", true, StringComparison.CurrentCultureIgnoreCase, true)]
+        [TestCase(" SS ", "�", true, StringComparison.CurrentCultureIgnoreCase, true)]
+        [TestCase(" ss ", "s", true, StringComparison.CurrentCultureIgnoreCase, true)]
+        [TestCase(" SS ", "s", true, StringComparison.CurrentCultureIgnoreCase, true)]
+#if !NETSTANDARD1_3            
```

Comment:
Looks like we need `!(NETSTANDARD!_3 || NETSTANDARD1_6)`. (You can test this on your own too by running `build -t test` at the command prompt from your local repo root.)